marathon/src/main/scala/io/buoyant/marathon/v2/AppIdNamer.scala

```diff
@@ -0,0 +1,119 @@
+package io.buoyant.marathon.v2
```

Comment:
the other namers don't package underneath the api version (eg v2)... I think the intent there is that AppIdNamer could work under any version of the api that's released